db/migration/1721137710001-RemoveHideLinesOutsideToleranceFromConfig.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class RemoveHideLinesOutsideToleranceFromConfig1721137710001
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`-- sql
+            UPDATE charts
+            SET config = JSON_REMOVE(config, '$."hideLinesOutsideTolerance"')
+            WHERE
+                type = "ScatterPlot"
+            AND
+                slug IN (
+                  "stunting-vs-level-of-prosperity-over-time",
+                  "growth-of-income-and-trade"
+                )
```

Comment:
It's better not to hard-code these slugs. Removing the property is the right thing to do in any case and there might be new charts created in the mean time between looking these up and the PR being merged.

Reply:
Fair enough, I decided to keep it tight in case of breaking something unrelated, but the opposite is the case, I'll make that change